test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -317,7 +317,7 @@ foo``u{2195}abc
 
 '@
 "@
-            $result | should be "`r`nfoo``u{2195}abc`r`n"
+            $result | should be "`nfoo``u{2195}abc`n"
```

Comment:
This doesn't look like an appropriate change for this PR - it's changing what is being tested.